openapi/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12-alpine as builder
+WORKDIR /moira-oas
+RUN npm install -g swagger-cli
+
+COPY . /moira-oas/
+RUN swagger-cli bundle main.yml -o build/openapi.yml -t yaml
+
+FROM swaggerapi/swagger-ui:latest
+COPY --from=builder /moira-oas/build/openapi.yml /openapi.yml
+ENV SWAGGER_JSON=/openapi.yml
```

Comment:
Probably `SWAGGER_YAML` will be better for this variable? Or this is reserved by swagger name?

Reply:
Yeah, it's reserved by the swagger docker image. Though it seems to work with both json and yaml files.